kythe/go/extractors/config/runextractor/compdb/compdb.go

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright 2018 The Kythe Authors. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Package compdb contains functionality necessary for extracting from a `compile_commands.json` file.
```

Comment:
No \`annotations\` in Go comments, please.

Reply:
(though it is fine and recommended to line-wrap comments to something approximating a screenful)